src/components/timeline/line.driver.ts

```diff
@@ -0,0 +1,39 @@
+import {StyleSheet} from 'react-native';
+import {ComponentProps} from '../../testkit/new/Component.driver';
+import {ViewDriver} from '../view/View.driver.new';
+
+export const LineDriver = (props: ComponentProps) => {
+  const lineDriver = ViewDriver({
+    renderTree: props.renderTree,
+    testID: `${props.testID}`
+  });
+
+  const entryPointDriver = ViewDriver({
+    renderTree: props.renderTree,
+    testID: `${props.testID}.entryPoint`
+  });
+
+  const getLine = () => {
+    const exists = (): boolean => {
+      return lineDriver.exists();
+    };
+    const getStyle = () => {
+      return StyleSheet.flatten(lineDriver.getElement().props.style);
+    };
+    const isVisible = (): boolean => {
+      const height = getStyle().height;
```

Comment:
I think getStyle() can return undefined

Reply:
StyleSheet.flatten can handle `undefined`, if `undefined` was passed it will return empty object.
```
const flattenedStyle = StyleSheet.flatten(undefined);
// flattenedStyle will be: {}
```

Reply:
```suggestion
    const isVisible = (): boolean => {
      if (exists()) {
        const height = getStyle()?.height;
        return (height ?? 0) > 0;
      }
      return false;
    };
```